src/translator/request.h

```diff
@@ -3,12 +3,18 @@
 
 #include <cassert>
 #include <future>
+#include <iostream>
```

Comment:
Are those used or are they leftover from previous implementation?

Reply:
I think the serializing for `Value = ProcessedRequestSentence` happens with ostringstream. We could change this though because the size of a `ProcessedRequestSentence` is known at start.

Reply:
Headers should be included in the file that needs them